spec/views/casa_cases/index.html.erb_spec.rb

```diff
@@ -42,17 +42,19 @@
   end
 
   it "Filters active/inactive casa_cases" do
-    active_case = create(:casa_case, active: true, casa_org: organization)
+    active_case = create(:casa_case, active: true, casa_org: organization, case_number: "CINA-1")
+    active_case1 = create(:casa_case, active: true, casa_org: organization, case_number: "CINA-2")
```

Comment:
@metamoni I think you had to add the case number due to the newish case-prefix filter:

<img width="411" alt="Screen Shot 2020-11-11 at 2 52 01 PM" src="https://user-images.githubusercontent.com/1938665/98873554-7d585000-242d-11eb-9496-d485de53f71d.png">

Would you mind adjusting the `casa_case` factory so that the case_number matches the `CINA-XXX` format so others don't run into the same issue you did?